docs/user-guide/gpus.md

```diff
@@ -0,0 +1,136 @@
+---
+assignees:
+- vishh
+title: GPU Support
+---
+
+Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes.
+This page describes how users can consume GPUs and the current limitations.
+
+## Pre-requisites
+
+1. Kubernetes nodes have to be pre-installed with Nvidia drivers. Kubelet will not detect Nvidia GPUs otherwise.
+2. A special **alpha** feature gate `Accelerators` has to be set to true across the system: `--feature-gates="Accelerators=true"`.
+3. Nodes must be using `docker engine` as the container runtime.
+
+The nodes will automatically discover and expose all Nvidia GPUs as a schedulable resource.
+
+## API
+
+Nvidia GPUs can be consumed via container level resource requirements using the resource name `alpha.kubernetes.io/nvidia-gpu`.
+
+```yaml
+apiVersion: v1
+kind: pod
+spec: 
+  containers: 
+    - 
+      name: gpu-container-1
+      resources: 
+        limits: 
+          alpha.kubernetes.io/nvidia-gpu: 2
```

Comment:
How does this expose the GPU in a Pod?  Give a command with kubectl exec that will show the GPU is accessible by the container.